text/0000-union-initialization-and-drop.md

```diff
@@ -0,0 +1,365 @@
+- Feature Name: union_initialization_and_drop
+- Start Date: 2018-08-03
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Unions do not allow fields of types that require drop glue, but they may still
```

Comment:
Nit: "drop glue" is such a rustc-specific jargon, perhaps "trivial ~destructor~ drop", or "drop not running any code", or something like this could be better.

Reply:
I think "(doesn't) need(s) drop" or "noop drop" could also work as terms.

Reply:
Don't we have a term for "the code that runs when something is dropped"? I feel that is not such a strange rustc-specific concept, all Rust implementations and also e.g. C++ have it.